src/components/PopoverMenu.tsx

```diff
@@ -138,6 +142,7 @@ function PopoverMenu({
     const [enteredSubMenuIndexes, setEnteredSubMenuIndexes] = useState<readonly number[]>(CONST.EMPTY_ARRAY);
 
     const [focusedIndex, setFocusedIndex] = useArrowKeyFocusManager({initialFocusedIndex: currentMenuItemsFocusedIndex, maxIndex: currentMenuItems.length - 1, isActive: isVisible});
+    const WrapComponent = shouldUseScrollView ? ScrollView : View;
```

Comment:
Use `Fragment` instead of `View`

Reply:
@s77rt On the latest main, we wrap all components including the header inside the scroll view. What do you think? I think we should only wrap the list item in the scroll view to keep the header is always at the top.

Reply:
Let's use that scrollview but move it not to wrap the title. Then please test both: https://github.com/Expensify/App/pull/48566#issuecomment-2340878728 and https://github.com/Expensify/App/issues/49217 and verify its working as expected

Reply:
Tested both case and it works well.